### ISSUE TITLE
No detection of Person, every other class is fine

### ISSUE THREAD

**Issue by freund-masasana** [ultralytics/ultralytics#9997]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Train, Val, Predict

### Bug

So I have a weird one here.. I start my training from the pretrained yolov8x.pt. The class Person is fine at this point. now I start the fine-tuning and with each epoch the class Person gets worse and worse to a point where after 200 epochs no person will be detected. All other classes are fine and detecting as expected. Persons are only detected when rotated (which we found out by having one randomly rotated image in the dataset). 
There is another case where bicycles get detected as Person. They should be a different class Vehicle, but I suspect this is a different issue. I checked the translation script and all bicycles are converted to vehicle, but maybe I need to check the raw data again for this.

I checked the labels.jpg (attached) for the train data and it seems Person is not corrupt as they are by far the class with the highest attendance (as expected). 
![labels](https://github.com/ultralytics/ultralytics/assets/157729735/b306ea63-ec59-4419-a957-83cad3b5dc62)

I also started a 1 epoch training with validation data as train data to get the same graphics and they seem fine also. 
![labels](https://github.com/ultralytics/ultralytics/assets/157729735/9c002d18-710b-487d-909e-745e283962d8)

After 1 epoch of training on the actual train data with the actual validation data, this person hits pretty good results
![0e83325c64a2dce105a1b95f750355cb jpg 0](https://github.com/ultralytics/ultralytics/assets/157729735/ec989cbf-3cfc-4ebc-b847-0ca48d10690c)

But after 5 epochs of training, the model doesn't recognise the person that well anymore
![0e83325c64a2dce105a1b95f750355cb jpg 0](https://github.com/ultralytics/ultralytics/assets/157729735/f7a287ee-017f-40e6-aa7e-61f80fd6e4a3)

Unless... 
![0e83325c64a2dce105a1b95f750355cb jpg 1](https://github.com/ultralytics/ultralytics/assets/157729735/13b707ea-822f-4829-9d4b-90f8ecb9e57c)

I checked the label files and the images. Everything seems ok and even in the preview batch images, persons are labeled correctly. There is almost no rotated pictures in the whole dataset (I found 1), so where is the problem? I don't know if this is actually a YOLO problem or not but I am all out of ideas. So any help is appreciated! I can also provide more info if needed.

Cheers

### Environment

Ultralytics YOLOv8.0.126 🚀 Python-3.10.12 torch-2.0.1+cu117 CUDA:0 (NVIDIA GeForce RTX 3080 Ti, 12035MiB)
Setup complete ✅ (12 CPUs, 31.2 GB RAM, 1766.8/1785.0 GB disk)

### Additional

The checks are from my machine. The main bulk of training were done on a cloud machine that is shut down now. But since the examples are generated on my machine and the problems persisted throughout, I thought it is ok to do so.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
Hey there 👋,

Thanks for the detailed explanation of your issue! It certainly sounds puzzling but let's see if we can help you troubleshoot this. From your description, it sounds like the model's ability to detect the "Person" class is degrading over epochs during fine-tuning, which is indeed unusual. Here are a few things you might consider:

1. **Overfitting:** It's possible that the model might be overfitting on non-person classes or specific kinds of backgrounds/contexts which do not include persons as frequently. Although it seems counterintuitive as "Person" is the most attended class, overfitting can sometimes manifest in unexpected ways.

2. **Learning Rate Decay:** If the learning rate is not being reduced over epochs, the model might be "forgetting" what it learned about the "Person" class. Ensure you're employing a learning rate scheduler that reduces the rate over time.

3. **Data Augmentation:** Since the issue seems to be mitigated with rotated images, it might help to introduce more varied data augmentation strategies, especially focusing on augmentations that modify object orientation and scale.

4. **Class Weight Balancing:** Though "Person" is the most present class, it might still be beneficial to experiment with class weighting to see if emphasizing the "Person" class more prominently during training helps.

5. **Review Data Labels:** Though you've checked, it might be worth revisiting the labeling to ensure no systemic biases or errors, particularly focusing on whether "Person" instances in various orientations are consistently well-labeled.

If you haven't already, you might also explore training with a different base model or adjusting the architecture to see if a particular configuration is more susceptible to this issue.

Here's a quick example of how you might introduce more aggressive data augmentation in your training setup:

```python
# Add data augmentation
augmentations = {
    'rotation': 15,  # degrees
    'translate': 0.1,  # fraction of total image width/height
    'scale': 0.1,  # scale augmentations
    'shear': 0  # shear augmentation
}

model.train(data='coco128.yaml', epochs=100, imgsz=640, augmentations=augmentations)
```

Keep in mind, the specifics will depend on your current setup and how you're invoking the training process. 

Don't hesitate to share more details or further observations. We're here to help 🚀. 

Best regards!
